js/lib/comments.js

```diff
@@ -113,6 +113,10 @@ elgg.comments.init = function() {
 		dc.toggleEdit();
 		return false;
 	});
+
+	// Highlight a comment if its id is found from the URL
+	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
+	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
```

Comment:
I would prefer toggling a class and using CSS transition to change the BG
color.
On Mar 9, 2015 5:39 PM, "Juho Jaakkola" <notifications@github.com> wrote:

> In js/lib/comments.js
> <https://github.com/Elgg/Elgg/pull/8050#discussion_r26051826>:
>
> > @@ -113,6 +113,10 @@ elgg.comments.init = function() {
> >  		dc.toggleEdit();
> >  		return false;
> >  	});
> > +
> > +	// Highlight a comment if its id is found from the URL
> > +	var commentId = elgg.getSelectorFromUrlFragment(document.URL);
> > +	$(commentId).addClass('elgg-highlight').delay(1000).removeClass('elgg-highlight', {duration: 1000});
>
> At first I though it wasn't even possible, but it ended up being a regular
> fade-away animation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Elgg/Elgg/pull/8050/files#r26051826>.
>